app/classifier/tasks/dropdown/index.cjsx

```diff
@@ -121,24 +120,30 @@ module.exports = React.createClass
   handleFocus: ->
     @menus[0].focus()
 
-  getOptions: (i, optionsKeys) ->
-    optionsKey = if optionsKeys.length then optionsKeys.join(';') else '*'
+  getOptionsKey: (i, optionsKey = '') ->
+    {selects} = @props.task
+    select = selects[i]
+    [condition] = selects.filter (filterSelect) => filterSelect.id is select.condition
```

Comment:
Hmmm, I got rid of that `optionKeys` variable because it didn't always stay in synch with `state.selectedOptions`. Anyone working on this code would have to somehow know that updating one of those variables should also update the other. A lookup table that stored the selected options keyed by `select.id` would be ideal here.